src/pages/MorePage.tsx

```diff
@@ -16,25 +16,54 @@ var style: CSSProperties = {
 
 const link = "h-16 text-2xl text-center align-bottom";
 
+export interface Recommend {
+  appstoreUrl: string;
+  labelDE: string;
+  labelEN: string;
+  playstoreUrl: string;
+}
+
 export const MorePage: FC<{ nextIssueId: number }> = ({nextIssueId}) => {
 
+  const [recommend, setRecommend] = useState<(Recommend)>();
+
   const handlePrev = () => {
     window.location.href = `/issue/${nextIssueId}`;
   }
+  
+  function getStoreLink(): string {
+    return  Capacitor.getPlatform() === "ios" ? recommend?.appstoreUrl!: recommend?.playstoreUrl!;
+  }
 
   async function openBrowser(url: string) {
     await Browser.open({url});
   }
 
+  const handleShare = async () => {
+    const lang = await getStoredLang()
+    try {
+      await Share.share({
+        title: lang === 'en' ? recommend?.labelEN : recommend?.labelDE ,
+        text: valFor(lang, 'share.text'),
+        url: getStoreLink(),
+        dialogTitle: lang === 'en' ? recommend?.labelEN : recommend?.labelDE 
+      })
+
+    } catch (e) {
+      // add Toast Message or somethng else here
+      console.error(typeof e);
```

Comment:
```suggestion
      console.error(e);
```
we should know the error thrown.